docs/oci-storage.md

```diff
@@ -1,40 +1,152 @@
 # OCI Storage
 
-OCI storage is an experimental feature to store a bundle's resources in an OCI registry, instead of k8s resources.
-Fleet won't be limited by [`etcd` size limitations](https://etcd.io/docs/v3.4/dev-guide/limit/). However,
-Fleet needs write access to a registry and that registry needs to be accessible by the agents in downstream clusters.
+Fleet stores Kubernetes bundle resources in etcd twice by default. This is done using the K8s API. However, etcd has storage limits and is not optimized for large workloads. As your deployments scale, it is more efficient to store these resources externally in an OCI (Open Container Initiative) registry.
 
-## Summary
+:::note
+Fleet recommends you to compress and encode the bundle into base64 to reduce the resource size.
+:::
 
-Fleet stores by default the bundle resources in etcd twice. This is done via the k8s API and there is a size limit, depending on the etcd configuration.
+If your bundle resource is greater than X MB(size), then it would be a better alternative to switch to OCI Registry as Storage. 
```

Comment:
How much is X?
I would, maybe, mention X as the limits of `etcd` configured for that particular cluster..